src/Common/src/Interop/Windows/HttpApi/Interop.HttpApi.cs

```diff
@@ -0,0 +1,1021 @@
+                HttpHeaderContentType = 12,   // entity-header  [section 7.1]
+                HttpHeaderContentEncoding = 13,   // entity-header  [section 7.1]
+                HttpHeaderContentLanguage = 14,   // entity-header  [section 7.1]
+                HttpHeaderContentLocation = 15,   // entity-header  [section 7.1]
+                HttpHeaderContentMd5 = 16,   // entity-header  [section 7.1]
+                HttpHeaderContentRange = 17,   // entity-header  [section 7.1]
+                HttpHeaderExpires = 18,   // entity-header  [section 7.1]
+                HttpHeaderLastModified = 19,   // entity-header  [section 7.1]
+
+
+                // Response Headers
+
+                HttpHeaderAcceptRanges = 20,   // response-header [section 6.2]
+                HttpHeaderAge = 21,   // response-header [section 6.2]
+                HttpHeaderEtag = 22,   // response-header [section 6.2]
+                HttpHeaderLocation = 23,   // response-header [section 6.2]
+                HttpHeaderProxyAuthenticate = 24,   // response-header [section 6.2]
+                HttpHeaderRetryAfter = 25,   // response-header [section 6.2]
+                HttpHeaderServer = 26,   // response-header [section 6.2]
+                HttpHeaderSetCookie = 27,   // response-header [not in rfc]
+                HttpHeaderVary = 28,   // response-header [section 6.2]
+                HttpHeaderWwwAuthenticate = 29,   // response-header [section 6.2]
+
+                HttpHeaderResponseMaximum = 30,
+
+
+                HttpHeaderMaximum = 41
+            }
+
+            private static string[] s_strings = {
+                    "Cache-Control",
+                    "Connection",
+                    "Date",
+                    "Keep-Alive",
+                    "Pragma",
+                    "Trailer",
+                    "Transfer-Encoding",
+                    "Upgrade",
+                    "Via",
+                    "Warning",
+
+                    "Allow",
+                    "Content-Length",
+                    "Content-Type",
+                    "Content-Encoding",
+                    "Content-Language",
+                    "Content-Location",
+                    "Content-MD5",
+                    "Content-Range",
+                    "Expires",
+                    "Last-Modified",
+
+                    "Accept-Ranges",
+                    "Age",
+                    "ETag",
+                    "Location",
+                    "Proxy-Authenticate",
+                    "Retry-After",
+                    "Server",
+                    "Set-Cookie",
+                    "Vary",
+                    "WWW-Authenticate",
+                };
+        }
+
+        private static string GetKnownHeader(HTTP_REQUEST* request, long fixup, int headerIndex)
+        {
+            if (NetEventSource.IsEnabled)
+            {
+                NetEventSource.Enter(null);
+            }
```

Comment:
Nit: elsewhere for cases like this I tried to keep it all on one line... while that goes against general bracing style elsewhere, I think it makes sense for these logging operations, where the ```if (NetEventSource.IsEnabled) ``` prefix is essentially just part of the method call.